### PR TITLE
Final french translation

### PR DESCRIPTION
**Pull request by aishenri**
Please just pull the french file in ts/ dir
The other one is a mistake of mine and I can't delete it

### CODE REVIEW

**Comment by vrruiz** [comment]
Sorry, but seems like a mix of Spanish, French and Portuguese translations.

File: ts/visualino_fr-fr.ts [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+    </message>
+    <message>
+        <location filename="../src/mainwindow.cpp" line="428"/>
+        <source>Done saving.</source>
+        <translation>Gardado finalizado.</translation>
+    </message>
+    <message>
+        <location filename="../src/mainwindow.cpp" line="475"/>
+        <source>Please, restart the application to display the selected language.</source>
+        <translation>Reinicie a aplicación para mostrar o idioma seleccionado.</translation>
```
Comment (vrruiz) [inline ts/visualino_fr-fr.ts]: This is not French.

File: ts/visualino_fr-fr.ts [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+    </message>
+    <message>
+        <location filename="../src/mainwindow.cpp" line="576"/>
+        <source>Building...</source>
+        <translation>Compilando...</translation>
+    </message>
+    <message>
+        <location filename="../src/mainwindow.cpp" line="850"/>
+        <source>There are unsaved changes that could be lost. Do you want to save them before continuing?</source>
+        <translation>Hai cambios sen gravar que podería perder. Desexa gardarlos antes de continuar?</translation>
```
Comment (vrruiz) [inline ts/visualino_fr-fr.ts]: This is not French.

**Comment by aishenri** [comment]
I did it with qt linguist I am surprised I didn't get this in french ?

Is there a template ? us or gb ?

By the way did you delete the fr file in root ? I am not expert and I 
couldn't delete it ?

It's strange because in my fork I have a french version, I will look 
deeper to see if I didn't forget something.

Sorry for the inconvenience.

**Comment by aishenri** [comment]
Apparently I got back the original file ?

Everything is to translate again

**Comment by aishenri** [comment]
I delete the files and uploaded and verified the french file... Can you get it.

**Comment by vrruiz** [comment]
Great. The last version looks fine!!

**Comment by aishenri** [comment]
Ouf !

How to translate the whole  block ?

**Comment by vrruiz** [comment]
To translate the blocks to French see the forum. I replied there.

**Pull request merged**